config/tests/jobs/jobs_test.go

```diff
@@ -1086,25 +1086,96 @@ func TestSigReleaseMasterBlockingOrInformingJobsShouldUseFastBuilds(t *testing.T
 	}
 }
 
+// extractUsesCIBucket returns true if kubetest --extract=foo
+// would use the value of --extract-ci-bucket, false otherwise
+func extractUsesCIBucket(extract string) bool {
+	if strings.HasPrefix(extract, "ci/") || strings.HasPrefix(extract, "gci/") {
+		return true
+	}
+	re := regexp.MustCompile(`^(v\d+\.\d+\.\d+[\w.\-+]*)$`)
```

Comment:
nit: I always find it really useful to have the Regex as vars at the top with a minimally descriptive name.